TestBangazonAPI/DatabaseFixture.cs

```diff
@@ -1,10 +1,74 @@
-﻿using System;
+﻿using BangazonAPI.Models;
+using Microsoft.Extensions.Configuration;
+using System;
 using System.Collections.Generic;
+using System.Data.SqlClient;
 using System.Text;
 
 namespace TestBangazonAPI
 {
-    class DatabaseFixture
+
+    public class DatabaseFixture : IDisposable
     {
+        private readonly string ConnectionString = @$"Server=localhost\SQLEXPRESS;Database=BangazonAPI;Trusted_Connection=True;";
+        public PaymentType AddDelTestType { get; set; }
+        public PaymentType EditType { get; set; }
+
+        public DatabaseFixture()
+        {
+            PaymentType newType = new PaymentType
+            {
+                AcctNumber = "9876543210",
+                Name = "TEST",
+                CustomerId = 2
+            };
+
+            PaymentType editType = new PaymentType
+            {
+                AcctNumber = "6789012345",
+                Name = "FOREDIT",
+                CustomerId = 2
+            };
+
+            using (SqlConnection conn = new SqlConnection(ConnectionString))
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @$"INSERT INTO PaymentType (AcctNumber, Name, CustomerId)
+                                        OUTPUT INSERTED.Id
+                                        VALUES ('{newType.AcctNumber}', '{newType.Name}', '{newType.CustomerId}')";
+                    int newId = (int)cmd.ExecuteScalar();
+
+                    newType.Id = newId;
+
+                    AddDelTestType = newType;
+
+                    cmd.CommandText = @$"INSERT INTO PaymentType (AcctNumber, Name, CustomerId)
+                                        OUTPUT INSERTED.Id
+                                        VALUES ('{editType.AcctNumber}', '{editType.Name}', '{editType.CustomerId}')";
+                    int newEditId = (int)cmd.ExecuteScalar();
+
+                    editType.Id = newEditId;
+
+                    EditType = editType;
+
+                }
+            }
+        }
+        public void Dispose()
+        {
+            using (SqlConnection conn = new SqlConnection(ConnectionString))
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @$"DELETE FROM PaymentType WHERE Name='TEST'";
```

Comment:
This will clean up the entry you created to test getting and deleting (i.e. the `newType` variable) but it will leave a few test payment types in your database. You can either:
1. edit this delete statement so that it deletes everything (I'd use the SQL `OR` clause I think)
2. Edit the names of your dummy data so that they all end up with the same name. For example, you could make sure that all of your integration test dummy data is named "integration test payment type" and then delete everything with a name of "integration test payment type"

Reply:
@Shaverp3 this is the only one that needs to be dealt with before a merge 🚀